driver/bpf/fillers.h

```diff
@@ -325,9 +325,8 @@ FILLER(sys_single, true)
 	int res;
 
 	val = bpf_syscall_get_argument(data, 0);
-	res = bpf_val_to_ring(data, val);
-
-	return res;
+	// TODO HOW TO avoid using bpf_val_to_ring?
+	return bpf_val_to_ring(data, val);
```

Comment:
TODO

Reply:
It's not so easy. We have 3 events that use `sys_single` filler:
* PPME_SYSCALL_FSTAT64_E -> takes a `"fd", PT_FD` param
* PPME_SYSCALL_UNLINK_E -> takes `"path", PT_FSPATH` param (moreover, it is an EF_OLD_VERSION event too!)
* PPME_SYSCALL_SEMOP_E -> takes `"semid", PT_INT32` param

So, if we really want to fix this, most natural way is to just get rid of `sys_single` and implement the 3 fillers.

Reply:
Gotcha. In that case yeah, let's just implement these fillers manually :)